plugins/main/public/components/overview/threat-hunting/dashboard/dashboard.tsx

```diff
@@ -19,6 +23,7 @@ import {
   EuiFlyoutHeader,
   EuiTitle,
   EuiButtonEmpty,
+  EuiLink,
```

Comment:
If not used, it could be deleted

Reply:
Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/6831/commits/42168192f7c168920b7b6cf111b812f32b894094